### PR TITLE
header, footer and about

### PR DESCRIPTION
**Pull request by blaizeliebs**
This inc header, footer and setup of the about page may need a few little tweaks. mostly structure and placement

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -1,20 +1,57 @@
...
+  return (
+    <div className={classes.root}>
+      <Container component='div'>
+        <AppBar position="static" color="transparent" className={classes.appBarHeader}>
+          <Toolbar>
+            <div className={classes.logo}>
+              <Image />
+            </div>
+            <div className={classes.nav}>
+              <Link to="/">
```
Comment (louMoxy) [inline src/components/header.js]: Can we add an `activeStyle` to there links so we can show the user the link they are currently on. 

Also the Link component creates an '<a>' tag which then is wrapping around the '<button>' tag which isn't valid HTML, can we change the button to be just a 'p' tag?

**Comment by louMoxy** [comment]
One little comment, but otherwise looks great 👍

File: src/components/header.js (hunk 1 above)
Comment (blaizeliebs) [reply src/components/header.js]: ah yes activeStyle is a good point did not think. about it.

and with you tag point this is a very valid point and will def change this, will just check how to get that material-ui button effect on the p tag! unless you don't think it is necessary.

We get on these changes a little later sorry for the delay.

**Comment by blaizeliebs** [comment]
Updated Merge request.

**Comment by louMoxy** [comment]
Awesome stuff! 😄

**Pull request merged**